src/pages/find-offers/FindOffers.jsx

```diff
@@ -50,10 +56,14 @@ const FindOffers = () => {
           open={ openFilters }
           showingTutorOffers={ showingTutorOffers }
         /> 
-        <OfferCard
-          offer={ mockOffer }
-          onBookmarkClick={ () => onBookmarkClick(mockOffer.id) }
-        />
+        <Box>
```

Comment:
it should be a separate container that will take a list of items and show them as a list or grid

Reply:
fixed